controllers/scripts/ipfs.go

```diff
@@ -26,6 +24,11 @@ user=ipfs
 # This is a custom entrypoint for k8s designed to run ipfs nodes in an appropriate
 # setup for production scenarios.
 
+INDEX="${HOSTNAME##*-}"
+
+PRIVATE_KEY=$(cat "/node-data/privateKey-${INDEX}")
+PEER_ID=$(cat "/node-data/peerID-${INDEX}")
```

Comment:
is there a reason why we wouldn't store these in /data/ipfs (or data) instead of having a new folder at the root?

Reply:
I see this is a separate volume, but then we can call it `secrets` or something.

I guess it is not possible to limit the secrets that a node can see to just be their key/value, is it?

Reply:
No sadly, that's one of the limitations of this approach. If one of the nodes becomes compromised, it also compromises all of the other identities.

We also considered setting them as environment variables, but since we're using a statefulset in this case, we're not able to configure each container with its own separate environment variable.